google/cloud/security/inventory/inventory_loader.py

```diff
@@ -284,7 +292,12 @@ def main(_):
     configs = FLAGS.FlagValuesDict()
 
     try:
-        pipelines = _build_pipelines(cycle_timestamp, configs, dao)
+        pipelines = _build_pipelines(
+            cycle_timestamp,
+            configs,
+            dao=dao,
+            project_dao=project_dao,
+            organization_dao=organization_dao)
```

Comment:
nit: Since you are already doing this, do you also want to do the same clean-up for group_dao here?  It's okay if rather not.  :)